integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/steps/database/ws/CoreApplicationKeyConfigurationRepository.java

```diff
@@ -12,6 +12,6 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.ApplicationKeyConfigurationRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsCoreApplicationKeyConfigurationRepository")
```

Comment:
Heb je overwogen om de naam van de class WsCoreApplicationKeyConfigurationRepository te maken (dit geldt voor meerdere classes in dit package)

Reply:
I have renamed this class and the other repository classes as well.